.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
```

Comment:
We can add a section `**Current behaviour**` this would make it much clear on what was expected and what's happening and another section called `**How frequently reproducible?**` to know how many times the issue as hit.

Reply:
`Current Behavior` is documented in the first step `Describe the bug`. This should clearly describe what is not working as intended and should mention how often the bug occurs with the `Steps to reproduce` documented in the next section

Reply:
My only reason of asking to have explicit sections is that it would prompt the bug reporter to fill the details, which can be missed in the current one.

Feel free to ignore this request if you feel it's not a good idea.

Reply:
Your ideas are good. The same info is requested in the current version of the PR with phrasing that is different than your suggestion.